Bibliotecas/Gym/Gym.ipynb

```diff
@@ -0,0 +1,2294 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Tutorial - Gym\n",
+    "\n",
+    "<center><img src=\"https://user-images.githubusercontent.com/48079888/93164280-23663300-f6e7-11ea-8cf9-e049d887a4ca.gif\" width=\"400\"/></center\n",
+    "\n",
+    "## Índice\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## O que é o Gym?\n",
+    "O **Gym** é um conjunto de ferramentas que ajudam no desenvolvimento e na comparação de algoritmos de aprendizado por reforço. A biblioteca é basicamente um conjunto de *environments*, ou ambientes de teste que o usuário pode utilizar pra testar seus algoritmos. Vale notar que esses ambientes têm uma interface compartilhada, desta forma torna-se possível escrever algoritmos gerais.\n",
+    "<img src=\"https://gym.openai.com/assets/dist/nav/nav-logo-dark-e5f2a4965e.svg\" width=\"400\"/>\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Como usar o Gym?\n",
+    "\n",
+    "Com o Gym, é possível testar e comparar algoritmos de aprendizado por reforço através dos diversos *environments* oferecidos pela biblioteca e as ferramentas de metrificação por ela implementadas, que possibilitam analisar o desempenho dos algoritmos utilizados."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Instalação\n",
+    "\n",
+    "A instalação do Gym pode ser realizada de duas formas diferentes, primeiramente através do pip (Python 3.5+):"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "source": [
+    "!pip install gym"
+   ],
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Defaulting to user installation because normal site-packages is not writeable\n",
+      "Requirement already satisfied: gym in /home/mths/.local/lib/python3.9/site-packages (0.18.3)\n",
+      "Requirement already satisfied: pyglet<=1.5.15,>=1.4.0 in /home/mths/.local/lib/python3.9/site-packages (from gym) (1.5.15)\n",
+      "Requirement already satisfied: numpy>=1.10.4 in /home/mths/.local/lib/python3.9/site-packages (from gym) (1.20.2)\n",
+      "Requirement already satisfied: scipy in /home/mths/.local/lib/python3.9/site-packages (from gym) (1.6.3)\n",
+      "Requirement already satisfied: cloudpickle<1.7.0,>=1.2.0 in /home/mths/.local/lib/python3.9/site-packages (from gym) (1.6.0)\n",
+      "Requirement already satisfied: Pillow<=8.2.0 in /home/mths/.local/lib/python3.9/site-packages (from gym) (8.2.0)\n"
+     ]
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Caso esteja acompanhando em um terminal, fora do notebook, retire o \"!\" antes de cada comando."
+   ],
+   "metadata": {}
+  },
```

Comment:
```suggestion
  {
   "cell_type": "markdown",
   "source": [
    "## Instalação\n",
    "\n",
    "A instalação do Gym pode ser realizada tanto pelo notebook quanto por um terminal, através do pip (Python 3.5+).\n"
    "Se estiver num notebook, rode a seguinte linha:"
   ],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "source": [
    "!pip install gym"
   ],
   "outputs": [
    {
     "output_type": "stream",
     "name": "stdout",
     "text": [
      "Defaulting to user installation because normal site-packages is not writeable\n",
      "Requirement already satisfied: gym in /home/mths/.local/lib/python3.9/site-packages (0.18.3)\n",
      "Requirement already satisfied: pyglet<=1.5.15,>=1.4.0 in /home/mths/.local/lib/python3.9/site-packages (from gym) (1.5.15)\n",
      "Requirement already satisfied: numpy>=1.10.4 in /home/mths/.local/lib/python3.9/site-packages (from gym) (1.20.2)\n",
      "Requirement already satisfied: scipy in /home/mths/.local/lib/python3.9/site-packages (from gym) (1.6.3)\n",
      "Requirement already satisfied: cloudpickle<1.7.0,>=1.2.0 in /home/mths/.local/lib/python3.9/site-packages (from gym) (1.6.0)\n",
      "Requirement already satisfied: Pillow<=8.2.0 in /home/mths/.local/lib/python3.9/site-packages (from gym) (8.2.0)\n"
     ]
    }
   ],
   "metadata": {}
  },
  {
   "cell_type": "markdown",
   "source": [
    "Caso esteja acompanhando em um terminal, fora do notebook, retire o \"!\" antes de cada comando."
   ],
   "metadata": {}
  },
```

Reply:
Marcando como resolvido e criando outro comentário apenas com as linhas 34 a 38.